src/index.js

```diff
@@ -23,10 +23,15 @@ export default class BodyBuilder {
 
   /**
    * Constructs the elasticsearch query body in its current state.
-   *
+   * @param  {String} version             Version to generate.
    * @returns {Object} Query body.
    */
-  build() {
+  build(version) {
+    if (!_.isUndefined(version) && version === 'v2') return this._buildV2();
+    return this._buildV1();
```

Comment:
ah, sure.. I unconsciously added semicolons. I'll check it and see wether I added it in other places as well :smile: